lib/core/boot/boot.ts

```diff
@@ -25,23 +26,26 @@ export const bootstrap = (): FluxifyServer => {
 	const options: Serve = {
 		port: config.stage === 'test' ? 0 : config.port,
 		development: config.stage === 'dev',
-		async fetch(request: FluxifyRequest): Promise<Response> {
-			request.id = randomUUID();
+		async fetch(request: FluxifyRequest, server: Server): Promise<Response> {
 			request.time = performance.now();
+			request.ip = server.requestIP(request)?.address ?? '';
```

Comment:
When does this happen? I only hope it's really rare... else some users get throttled real fast.

Reply:
When the connection get's closed or we are on a unix socket according to bun.

Reply:
Okay I hope It doesn't cause any issues :)